spec/inventory_source_spec.rb

```diff
@@ -7,7 +7,6 @@
   let(:raw_instance)        { build(:response_instance, :klass => described_class) }
 
   include_examples "Collection Methods"
-  include_examples "Crud Methods"
```

Comment:
Consistency with `activity_stream` - `inventory_sources` does not support `post`, `put` or `patch` so I think we should pull out the shared examples and implement CRUD on those 'special' endpoints specific to our needs.